mappings/net/minecraft/util/profiling/jfr/BaseJfrProfiler.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_6611 net/minecraft/util/profiling/jfr/BaseJfrProfiler
```

Comment:
Since this is an api class actually used by users, I would recommend calling this `FlightProfiler` or something that don't prompt people to use `JfrProfiler` impl class over this class.